### ISSUE TITLE
System Error when running Ganache 2.0.0 on linux

### ISSUE THREAD

**Issue by sarbogast** [trufflesuite/ganache#1168]
PLATFORM: linux
GANACHE VERSION: 2.0.0

I'm simply trying to open a simple workspace created with Ganache 2.0.0-beta.2

EXCEPTION:
```
Error: Could not find 'node'. NodeJS is required to be installed to link Truffle projects.
    at ChildProcess.child.on.error (/src/truffle-integration/projectDetails.js:65:17)
    at emitOne (events.js:115:13)
    at ChildProcess.emit (events.js:210:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
    at onErrorNT (internal/child_process.js:374:16)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by DiscRiskandBisque** [comment]
Good news @sarbogast! This issue has been addressed in the `2.0.0` release, please upgrade and let us know if the issue persists.

I should note we're aware of a similar issue on the Elementary OS distro (that _is_ present in `2.0.0`), which Linux distro are you using?

**Comment by sarbogast** [comment]
No no: I had this issue with version 2.0.0 on Ubuntu 18.04

**Comment by udartsev** [comment]
Ubuntu 16.04 same error

PLATFORM: linux
GANACHE VERSION: 2.0.0

EXCEPTION:
```
Error: watch /www/magnat-go_contracts/truffle-config.js ENOSPC
    at exports._errnoException (util.js:1024:11)
    at FSWatcher.start (fs.js:1394:19)
    at Object.fs.watch (fs.js:1420:11)
    at ProjectFsWatcher.start (/src/truffle-integration/projectFsWatcher.js:40:29)
    at new ProjectFsWatcher (/src/truffle-integration/projectFsWatcher.js:25:10)
    at ProjectsWatcher.add (/src/truffle-integration/projectsWatcher.js:130:23)
    at process.<anonymous> (/src/truffle-integration/index.js:59:34)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by davidmurdoch** [comment]
@udartsev, your issue looks to be related to https://stackoverflow.com/questions/22475849/node-js-what-is-enospc-error-and-how-to-solve/32600959#32600959 Can you try that out open a new issue if you can't find a solution in the Stack Overflow answers?

**Comment by davidmurdoch** [comment]
@sarbogast, what node version do you have installed?

**Comment by sarbogast** [comment]
I tried the commands suggested in the stackoverflow answer and I still have the same exact issue.
And I have NodeJS v10.15.3 installed

**Comment by Troublor** [comment]
@sarbogast How do you install node.js? 
I encountered this problem when I install node.js by downloading tar package and add `node` to PATH in `.bashrc`. I found later that `node` should be added to PATH in profile and there isn't such problem any more.

**Comment by sarbogast** [comment]
@Troublor  I installed NodeJS via NVM, because otherwise Truffle had issues with sudo permissions. But just as we discussed with the team back then, should it really matter how NodeJS is installed?

**Comment by Troublor** [comment]
@sarbogast In my case, it does relate to how NodeJS is installed. Have you tried open Ganache using `./Ganache-2.0.0.AppImage` in terminal?

**Comment by sarbogast** [comment]
Ganache launches fine, it's when I want to open a workspace I created before that I have this exception.

**Comment by Troublor** [comment]
@sarbogast Yes I was talking about opening a workspace from a Ganache opened by `./Ganache-2.0.0.AppImage`. If the problem still exists, I am afraid your problem is not due to the same cause as mine.

**Comment by sarbogast** [comment]
@Troublor what do you mean by "Ganache opened by ./Ganache-2.0.0.AppImage" exactly? Because I installed ganache by executing the AppImage file already.

**Comment by Troublor** [comment]
@sarbogast AppImage is not a installation file. It's an executable file, the Ganache itself. The command to run AppImage file is use `./Ganache-2.0.0.AppImage` in the terminal. 
Here's my logic. If a terminal session can find the command `node`, then the AppImage file executing in this terminal session should also be able to find `node`.

**Comment by davidmurdoch** [comment]
Possibly related to https://github.com/trufflesuite/ganache/pull/1166

**Comment by sarbogast** [comment]
Resolved in 2.0.1-beta.0
